docs/source/schemas/index.rst

```diff
@@ -133,33 +133,35 @@ characteristics, among others. The types are:
 +-------------------------------+--------------+---------------+----------------------+----------------+
 | NULLABLE\_FIXED\_LONG         | long         | byte[9]       | YES                  | YES            |
 +-------------------------------+--------------+---------------+----------------------+----------------+
-| VAR\_LONG                     | long         | byte[len]\*   | YES                  | YES            |
+| VAR\_LONG                     | long         | byte[len] \*  | YES                  | YES            |
++-------------------------------+--------------+---------------+----------------------+----------------+
+| VAR\_SIGNED\_LONG             | long         | byte[len] \*  | YES                  | YES            |
 +-------------------------------+--------------+---------------+----------------------+----------------+
-| VAR\_SIGNED\_LONG             | long         | byte[len\*]   | YES                  | YES            |
+| UUID                          | UUID         | byte[16]      | YES                  | YES            |
```

Comment:
I don't think so, it's just an implementation detail. The only case I can think of where a user might care about this is in the context of range scans, but we already specify that.